api/storage_service.cc

```diff
@@ -1697,8 +1697,19 @@ rest_repair_tablet(http_context& ctx, sharded<service::storage_service>& ss, std
         } else {
             tokens_variant = tokens;
         }
+        auto hosts = req->get_query_param("hosts_filter");
+        auto dcs = req->get_query_param("dcs_filter");
 
-        auto res = co_await ss.local().add_repair_tablet_request(table_id, tokens_variant);
+        std::unordered_set<sstring> hosts_vec;
+        boost::split(hosts_vec, hosts, boost::algorithm::is_any_of(","));
```

Comment:
We try to move away from `boost` ranges, isn't there a way to do this with `std::ranges`?